Adyen/UI/Form/Items/Value/Validatable/FormValidatableValueItemView.swift

```diff
@@ -103,9 +103,20 @@ open class FormValidatableValueItemView<ValueType, ItemType: FormValidatableValu
     
     private func hideAlertLabel(_ hidden: Bool, animated: Bool = true) {
```

Comment:
Not related to the changes, but maybe by renaming `hidden` to `shouldHide` or `hide` it makes the logic easier to read :)